include/LLGL/ImageFlags.h

```diff
@@ -61,6 +61,10 @@ struct ImageView
 
     //! Specifies the size (in bytes) of the image data. This is primarily used for compressed images and serves for robustness.
     std::size_t dataSize    = 0;
+
+    //! Specifies the size of one row of the image data.
+    //! When this is zero then the width of the texture region is used.
+    std::uint32_t stride = 0;
```

Comment:
Could you please rename this to `rowStride` so we can later add `layerStride` which can be translated to `GL_UNPACK_IMAGE_HEIGHT` in GL and D3D has `SrcDepthPitch` for instance. We can add this later, adding `rowStride` is difficult enough for 5 backends :)

Please also add a new parameter for this field to the constructor with a default value of 0.

Lastly, I don't see this implemented for Metal. I can help with that but until then, please change the comment to this:
```cpp
/**
\brief Specifies the size of one row of the image data.
\remarks When this is zero then the width of the texture region is used.
\note Only supported with: Direct3D 12, Direct3D 11, Vulkan, OpenGL.
*/
```